File: demoApiAngular/src/main/java/com/ducthang/demoApiAngular/model/Tour.java
package com.ducthang.demoApiAngular.model;

import lombok.Data;

import javax.persistence.*;

@Entity
@Data
public class Tour {
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    private Long id;

    private String name;
    private String image;
    private String price;
    private String description;

    @ManyToOne
    Category category;
}

File: BTVN_QL_Tour_Du_Lich/src/app/model/tour.ts
import {Category} from "./category";

export class Tour{
  id!: number;
  name!: string;
  image!: string;
  price!: number;
  description!: string
  category!: Category

  constructor(id: number, name: string,image: string ,price: number, description: string, category: Category) {
    this.id = id;
    this.name = name;
    this.image = image;
    this.price = price;
    this.description = description;
    this.category = category;
  }
}
